web_services/foxx/brightid/schemas.js

```diff
@@ -94,9 +94,25 @@ const operations = {
     sig: joi.string().required().description('deterministic json representation of operation object signed by the dismisser'),
   },
   'Add Admin': {
-    id: joi.string().required().description('brightid of the user who has admin rights in the group and can grant administratorship to other members'),
+    id: joi.string().required().description('brightid of one of the current admins of the group'),
     admin: joi.string().required().description('brightid of the member whom is being granted administratorship of the group'),
-    group: joi.string().required().description('the unique id of the group that new admin is being added to'),
+    group: joi.string().required().description('the unique id of the group'),
+    sig: joi.string().required().description('deterministic json representation of operation object signed by the admin user represented by id'),
+  },
+  'Update Group': {
+    id: joi.string().required().description('brightid of one of the admins of the group'),
+    group: joi.string().required().description('the unique id of the group'),
+    url: joi.string().required().description('the new url that group data (profile image and name) encrypted by group AES key can be fetched from'),
+    sig: joi.string().required().description('deterministic json representation of operation object signed by the user represented by id'),
+  },
+  'Add Subkey': {
+    id: joi.string().required().description('brightid of the user who is adding new subkey'),
+    subkey: joi.string().required().description('the public key of the new key pair that user can sign operations with'),
+    sig: joi.string().required().description('deterministic json representation of operation object signed by the user represented by id'),
+  },
+  'Remove Subkey': {
+    id: joi.string().required().description('brightid of the user who is removing the subkey'),
+    subkey: joi.string().description('the subkey that is being removed'),
```

Comment:
Instead of having an undocumented feature that leaving out subkey removes all subkeys, how about an extra boolean parameter called `all` that only works if `subkey` is undefined?

Reply:
The remove all subkeys (remove all signing keys) functionality is removed and we can handle it client-side by using the removeSigningKey function